.github/workflows/ant.yml

```diff
@@ -0,0 +1,67 @@
+name: Java CI
```

Comment:
This file seems to be different than the rest of the PR. Probably split into a separate PR

Reply:
I can remove the GitHub action if needed. I put it in so that following PRs could be shown to be working.

Reply:
I recommend still making use of GitHub actions on the repo, even if there's a CI build elsewhere. It's a lot easier to get an idea if pull requests are ok if pipelines are being run within GitHub to at least build & run tests with various JDK versions. Also it's externally visible. Right now I have no idea if Apache has run tests on this pr or not.